src/Features/LanguageServer/Protocol/Workspaces/LspWorkspaceManager.cs

```diff
@@ -168,15 +168,15 @@ public void UpdateTrackedDocument(Uri uri, SourceText newSourceText)
     /// 
     /// This is always called serially in the <see cref="RequestExecutionQueue"/> when creating the <see cref="RequestContext"/>.
     /// </summary>
-    public async Task<Document?> GetLspDocumentAsync(TextDocumentIdentifier textDocumentIdentifier, CancellationToken cancellationToken)
+    public async Task<(Workspace?, Document?)> GetLspDocumentAsync(TextDocumentIdentifier textDocumentIdentifier, CancellationToken cancellationToken)
```

Comment:
this is used for testing purposes (where tests validate the particular Workspace for a Document.  We want to ban going from Solution->Workspace, so for that testing purpose, we jsut return both.